packages/directory/addon/components/dir-asset-row-actions.js

```diff
@@ -24,9 +24,9 @@ export default Component.extend({
    * @property {String} type - the type of the asset
    */
   type: computed('value', function() {
-    let value = get(this, 'value');
+    let { value } = this;
```

Comment:
In the past destructures have failed for me when the 3.x addons have been used with 2.x apps.  Does this work with zig or teller?

Reply:
teller is already upgraded but I'll check with zig

Reply:
Ziggurat doesn't use directory anyway

Reply:
I've probably changed to this syntax in a few other places

Reply:
oh wait I forgot it's only a directory upgrade :laughing:

Reply:
Teller's looking good so far